src/SeleniumLibrary/keywords/webdrivertools.py

```diff
@@ -294,6 +294,18 @@ def active_drivers(self):
                 open_drivers.append(driver)
         return open_drivers
 
+    @property
+    def active_driver_ids(self):
+        open_driver_ids = []
+        for index, driver in enumerate(self._connections):
+            if driver not in self._closed:
+                open_driver_ids.append(index + 1)
+        return open_driver_ids
+
+    @property
+    def active_aliases(self):
```

Comment:
Instead of wrapping like this, it would be good to change `_alias` as a public method. `_alias` -> `alias`

Reply:
But _aliases is a field of the Class ConnectionCache, which is part of robot.utils...
Do you wand me to make a pull request to this? This would brake a lot of libraries.
Also SSH Library uses _aliases etc...

May be i completely misunderstood.

Reply:
I did not know that, no need to change.